arm_robots/launch/med.launch

```diff
@@ -0,0 +1,66 @@
+<launch>
+<!--    <env name="ROSCONSOLE_CONFIG_FILE"-->
+<!--         value="$(find arm_robots)/config/victor_rosconsole.conf"/>-->
+
+    <arg name="sim" default="false"/>
+    <arg name="gui" default="true"/>
+    <arg name="update_description" default="true"/>
+    <arg name="verbose" default="false"/>
+
+    <group if="$(arg update_description)">
+        <!-- uploads robot description and setups up other parameters -->
+        <include file="$(find iiwa_med_moveit_config)/launch/planning_context.launch">
+            <arg name="load_robot_description" value="true"/>
+        </include>
+    </group>
+
+
+    <group if="$(arg sim)">
+        <!-- Start gazebo -->
+        <group> <!-- this empty group tag creates a scope for the "remap" command -->
+            <!-- because we use ros_control only for the joints of the arms, not the grippers,
```

Comment:
arms -> arm
grippers -> gripper

Reply:
For now, we don't need this remapping, so I've removed the group/remap entirely.